great_expectations/datasource/data_connector/configured_asset_gcs_data_connector.py

```diff
@@ -72,7 +72,7 @@ def __init__(
         Args:
             name (str): required name for DataConnector
             datasource_name (str): required name for datasource
-            container (str): container name for Google Cloud Storage
+            bucket_or_name (str): bucket name for Google Cloud Storage
```

Comment:
possibly will change this but please disregard the `bucket` vs `bucket_or_name` naming argument in this PR. If we decide to make changes, the impact will be larger than just this file and should warrant a separate PR